core/modules/system/css/menu-dropdown.theme.css

```diff
@@ -48,14 +48,14 @@
   content: '-';
 }
 
-@media (min-width: 768px) {
+@media (min-width: 48em) {
```

Comment:
Yeup, just wanted to make sure it 'broke' at the same point that our layouts do.

I have everything on either:
25em, 34em, 48em, and 62em

25em doesn't actually show up in our layouts CSS, but I needed a smaller one, so there it is :stuck_out_tongue: